### PR TITLE
add move column

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexView.coffee [hunk 1]
```diff
@@ -96,11 +120,12 @@ class OpenOrchestra.AreaFlex.AreaFlexView extends OrchestraView
       subArea.order = $('.area-flex[data-area-id="'+area.area_id+'"]', @$el).index()
       data.areas.push(subArea)
     url = @options.area.get('links')._self_move_area
+    console.log data
```
Comment (Thiblef) [inline BackofficeBundle/Resources/public/coffee/areaFlex/AreaFlexView.coffee]: rm console.log

**Comment by Thiblef** [comment]
:+1:

**Comment by itkg-alavieille** [comment]
@Thiblef  pr updated

**Comment by Thiblef** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1: don't forget a OO tag in the comment if required

**Pull request merged**